### PR TITLE
feature: added rating

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added rating

### CODE REVIEW

**Review by kavyakunder — COMMENTED** [review]
Code looks good to me.

File: components/rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+          <iframe
+            src="https://www.thiscodeworks.com/embed/6200ea264dd44f0015f3cd94"
+            style="width: 100%; height: 177px"
+            frameborder="0"
+          ></iframe>
+        </div>
+      </section>
+    </div>
+
+    <script src="./rating.js"></script>
```
Comment (dev-enforced) [inline components/rating/rating.html]: Couldn't see its implementation on the main site. So is it required?You can remove it if not.

**Review by dev-enforced — APPROVED** [review]
Looks good to me . You can merge it Om. Make sure you pass a value inside the alt attribute of CRIMSON logo img later. Is there a need of the JS file which you mentioned below

File: components/rating/rating.html (hunk 1 above)
Comment (omgaikwad) [reply components/rating/rating.html]: I will remove it

**Pull request merged**